Source/public/views/home.html

```diff
@@ -2,14 +2,15 @@
 <div class="row w-100 m-0" >
 	<div class="col-sm-6 mx-auto text-center">
 	<h1 class="header">Oregon State University</h1>
-	<h5 class="header" >Campus Building Energy Usage Information</h5>
```

Comment:
if these classes "header, header2, header3" just have font family delcreations... why not just make classses for each font ".font-stratum"

Reply:
good idea